### ISSUE TITLE
Python tools not initialized because of missing .py suffix

### ISSUE THREAD

**Issue by linkous8** [returntocorp/bento#284]
## Overview
Semi-related to #208 

`bento init` misses Python executable files that didn't have .py extension (but have a python shebang). The tools were installed correctly once I renamed the file (e.g., from measure to measure.py)

## Current Behavior
Running `bento init` on a project which has no files with .py extensions (but have a python shebang) results in the following output:

```
Press ENTER to add Bento to this project: 

Creating default ignore file at .bentoignore․․․․․․․․․․․․․․․․․․․․․🍱 Done      p
Creating default configuration at .bento/config.yml․․․․․․․․․․․․․․🍱 Done      p
Enabling autorun (see $ bento enable autorun --help)․․․․․․․․․․․․․🍱 Done      p

⚠ Bento can't automatically identify this project. Please manually configure
this project:
  configure a tool manually․․․․․․․․․․․․․․․ $ bento enable tool TOOL
  list available tools․․․․․․․․․․․․․․․․․․․․ $ bento enable tool --help

Installing tools:

✘ No tools are configured in this project's config.yml. Please contact
support@r2c.dev if this is incorrect.
```

## Steps to Reproduce

1. Add a python shebang to a .py file, rename it to remove it's .py suffix (ensure there are no other .py files in the project)
2. Run `bento init`

## Possible Solution
Detect python shebang during init using similar logic to the resolution of #208

**Comment by dlukeomalley** [comment]
Thanks @linkous8 for the detailed ticket and referencing past tickets. I'm able to reproduce this on  Bento 0.9.1. We'll queue this up for today.

**Comment by dlukeomalley** [comment]
@linkous8 We cut a pre-release with a fix for this ticket: `bento/0.9.2b1`. Are you able to try it and confirm that your python files are detected now? 

```shell
$ pip3 install --upgrade --pre bento-cli
```

We'll cut a full release either later this morning or possibly Monday, just to avoid shipping new code before a weekend.

**Comment by linkous8** [comment]
@dlukeomalley I just confirmed bento init is detecting my python files on the pre-release version. Thanks a ton for the quick turnaround

**Comment by dlukeomalley** [comment]
These changes shipped in 0.10. Thanks again @linkous8 for filing the ticket.
